_resources/port1.0/group/python-1.0.tcl

```diff
@@ -306,14 +351,19 @@ proc python_get_defaults {var} {
             }
         }
         binary_suffix {
+            # TODO: pypy
```

Comment:
Is there still something that needs to be done here or does the comment just need to be deleted?

Reply:
Not any more, no.